math/src/circle/point.rs

```diff
@@ -1,82 +1,24 @@
-use super::errors::CircleError;
 use crate::field::traits::IsField;
 use crate::field::{
     element::FieldElement,
     fields::mersenne31::{extensions::Degree4ExtensionField, field::Mersenne31Field},
 };
-use core::ops::{Add, Mul};
+use core::ops::{Add, AddAssign, Mul, MulAssign};
 
 /// Given a Field F, we implement here the Group which consists of all the points (x, y) such as
 /// x in F, y in F and x^2 + y^2 = 1, i.e. the Circle. The operation of the group will have
 /// additive notation and is as follows:
-/// (a, b) + (c, d) = (a * c - b * d, a * d + b * c)
+/// (a, b) + (c, d) = (a * c - b * d, a * d + b * c).
+
 #[derive(Debug, Clone)]
 pub struct CirclePoint<F: IsField> {
     pub x: FieldElement<F>,
     pub y: FieldElement<F>,
 }
 
-impl<F: IsField + HasCircleParams<F>> CirclePoint<F> {
```

Comment:
Careful here, I think you've undone part of [e0fa390](https://github.com/lambdaclass/lambdaworks/pull/924/commits/e0fa3904aeb98a3146f4a5f73f05c4f801f76287)'s work

Reply:
You're right, thank you!